### PR TITLE
Delete a class date from the attendance sheet

### PR DESCRIPTION
**Pull request by hyunzeekim**
### Implementation description

- Clicking "Take Attendance" displays remove icon button above each date
- Clicking the remove button deletes the date column
- If "Done Attendance" is clicked, the deleted date column should not exist
- If "Done Attendance" is not clicked, the deleted date column should still exist

### Steps to test

1. Go to attendance table and click Take Attendance
2. Click the remove button and verify the column is removed! (if Done Attendance is not clicked, make sure the column still exists)

### What should reviewers focus on?

- correctness

### Checklist

- [X] My PR name is descriptive and in imperative tense
- [X] I have run the linter
- [X] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by Tayef-Shah** [comment]
just a general comment on this PR, let's try to refrain from writing sorta vague commit messages. For example "small change" could have been "change current date from heading to paragraph size font". Not a super big deal for this one, but it's helpful to know what kind of change happened, even the smallest ones :)

**Comment by hyunzeekim** [comment]
> just a general comment on this PR, let's try to refrain from writing sorta vague commit messages. For example "small change" could have been "change current date from heading to paragraph size font". Not a super big deal for this one, but it's helpful to know what kind of change happened, even the smallest ones :)

ah okay thanks!! will keep that in mind next time :)

File: src/components/sessions/session-detail-view/AttendanceTable.tsx [hunk 1]
```diff
@@ -228,6 +237,26 @@ const AttendanceTable = ({
                 }
               />
             ),
+            customHeadLabelRender: () => (
+              <div style={{ width: 60, textAlign: "center" }}>
```
Comment (Tayef-Shah) [inline src/components/sessions/session-detail-view/AttendanceTable.tsx:241]: the useStyles hook would make it easier to maintain, compared to inline styling

**Review by Tayef-Shah — COMMENTED** [review]
good work adding the aria-labels, just left a quick comment

File: src/components/sessions/session-detail-view/AttendanceTable.tsx (hunk 1 above)
Comment (hyunzeekim) [reply src/components/sessions/session-detail-view/AttendanceTable.tsx:241]: yep! sorry sorry brittany left that comment too but i forgot to fix that along with the aria-labels. will take a look!

**Review by brittanylau — APPROVED** [review]
nice!!! just one last comment about styling above, but everything else looks great :)

**Pull request merged**